Add fixes for banner size issues and Update Dashboard

Abban opened the pull request:
* Fixed size issue checker not firing
* Fixed name of size issue event

timEulitz commented on desktop/css/styles.pcss:
```diff
@@ -508,6 +507,10 @@
 				border-radius: 0 0 0.4em 0;
 			}
+			.select-group__custom-input:focus,
+			.select-group__custom-input:active {
```
You should move the `:focus` and `:active` blocks into the `.select-group__custom-input` block which is defined right above this one to reduce redundancy:

```
.select-group__custom-input {
    border-radius: 0 0 0.4em 0;
    &:focus, 
    &:active {
        //CSS here
}
```

timEulitz commented on desktop/css/styles_var.pcss:
```diff
@@ -84,3 +83,20 @@
+#WMDE_Banner-form {
+	.select-group__custom-input,
+	.select-group__custom-input:focus,
+	.select-group__custom-input:active {
```
It should be possible to do this without re-writing all rules for all three of those cases and using `important`, especially when working with a single ID selector. 

If there is an issue with getting rules applied to elements of this specific selector, you may want to change `styles.pcss` a bit so that it is easier to overwrite these values. I think the old CSS was not fully optimized in this regard so this may be a good opportunity to change that.

timEulitz commented on desktop/css/styles_var.pcss:
```diff
@@ -84,3 +83,20 @@
+	}
+
+	.euro-sign {
+		right: 1.8em !important;
```
Same as above, I think this should be possible without the use of `!important`.

gbirke commented on desktop/banner_var.js:
```diff
@@ -1,10 +1,8 @@
 require( './css/wlightbox.css' );
 
-// For A/B testing different styles, load
-require( './css/styles_var.pcss' );
```
In the past, we always kept this line here, together with the comment, either active or commented out, as a hint on what's happening. But that could also be seen as a bad practice. Let's discuss.

tzhelyazkova replied:
Let's remove this comment and when `styles_var.pcss` is deleted, then this require statement would also be deleted.

Abban committed:
Add fixes for banner size issues

* Fixed size issue checker not firing
* Fixed name of size issue event

Abban committed:
Add changes for Dandy tests

This incorporates the changes from
https://github.com/wmde/fundraising-banners/pull/111

Abban committed:
Update dashboard styles

Sperling-0 committed:
Add space between two columns

Sperling-0 committed:
Campaign central notice page opens in a new tab

Abban committed:
Fix entry point imports

Abban committed:
Fix close buttons

Abban committed:
Make banners in dashboard flex with gap

moiikana commented:
maybe it's possible to add a 
```
z-index: 2;
```
to `.header, .footer` in `dashboard/styles.scss`, to prevent the navigation header and footer from being in the background during scrolling

Abban committed:
Display current campaign at top of dash

moiikana committed:
Add type to parameter
